packages/manager/apps/web/client/app/css/less/common/wizard.less

```diff
@@ -21,6 +21,7 @@
   border-top-right-radius: 6px;
   font-size: 90%;
   left: 100%;
+  max-height: 0;
```

Comment:
Maybe instead we should think of uniformizing with other modals/pages ? What do you think ?

Reply:
This is used in a lot of places in `web`. I think we need to also add this component to `ui-kit` as well.

Reply:
Uniformizing part that came from `ng-ovh-utils`, since it's has been deprecated, is not necessary.

We should probably found an other solution regarding how wizard could be turned maybe in a stepper form for instance. It will facilitate how help message could be displayed.